sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -567,6 +567,9 @@ extern SYCL_EXTERNAL void
 __spirv_ocl_prefetch(const __attribute__((opencl_global)) char *Ptr,
                      size_t NumBytes) noexcept;
 
+__SYCL_CONVERGENT__ extern SYCL_EXTERNAL __SYCL_EXPORT bool
+    __spirv_GroupNonUniformElect(__spv::Scope::Flag) noexcept;
+
```

Comment:
I think, I removed this change from my patch as it wasn't needed

Reply:
Interesting. I wonder how it got left in here?  I double checked and you are right. `GroupNonUniformElect` is not in the current codebase and has no callers. I removed its declaration (and forward decl). Thanks!